openvdb/util/PagedArray.h

```diff
@@ -185,7 +186,7 @@ namespace util {
 
 template<typename ValueT,
          size_t Log2PageSize = 10UL,
-         template<typename ...> class TableT = std::vector>
+         template<typename ...> class TableT = std::deque>// must allow mutiple threads to call operator[] as long as only one thread calls push_back
```

Comment:
If it's not safe to use `std::vector`, would it not be wise to add a static assertion (ideally, one that invokes an `IsThreadsafeRWContainer` type trait)?

Or maybe this class doesn't need to be templated on `TableT` at all?

Reply:
removed last template argument